eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -7,6 +7,9 @@
     <UsagePattern IdentityGlob="Nuget.*/*" />
     <UsagePattern IdentityGlob="Microsoft.Build.NuGetSdkResolver/*" />
 
+    <!-- Sources for this package are included in-repo -->
+    <UsagePattern IdentityGlob="Microsoft.NET.Sdk.Android.Manifest-*/*" />
```

Comment:
I don't think this should be added here.  If it is in this repo, then it shouldn't be detected as a prebuilt.  I suspect something isn't working as expected.

Reply:
How does it "find" prebuilts? Would `<PackageDownload Include="Foo" Version="[1.0.0]"/>` trigger this?

Reply:
The package is being included as a download dependency:
![image](https://github.com/user-attachments/assets/67611fe8-ca0a-4c71-8414-61fb491d9ca4)

So in this case, yes, `<PackageDownload Include="Microsoft.NET.Sdk.Android.Manifest" .../>` is triggering the prebuilt.

Reply:
More often than not, prebuilts are caused by project references & transitive dependencies. Technically, [any package used in the build that is not built from source is a prebuilt](https://github.com/dotnet/source-build/blob/main/Documentation/eliminating-pre-builts.md#what-is-a-prebuilt), so it makes sense that a download dependency would cause a prebuilt, but this is the first time I've seen a package download cause a prebuilt.